parsers/julia.c

```diff
@@ -1180,7 +1253,6 @@ static void parseImport (lexerState *lexer, vString *scope, int parent_kind)
             advanceToken(lexer, true);
         }
     }
-
```

Comment:
Understandable. However, this deletion is nothing to do with generating reftags.
So
(1) don't delete thelin in this commit,
(2) (optional) make another commit for deletion.

Reply:
Done.